commcare_connect/events/views.py

```diff
@@ -34,9 +35,24 @@ def create(self, request, *args, **kwargs):
 
         serializer = self.get_serializer(data=request.data, many=True)
         serializer.is_valid(raise_exception=True)
-
-        event_objects = [Event(**item) for item in serializer.validated_data]
-        Event.objects.bulk_create(event_objects)
+        try:
+            event_objects = [Event(**item) for item in serializer.validated_data]
+            Event.objects.bulk_create(event_objects)
+        except Exception as e:
+            # Bulk create failed, try saving each item individually
+            failed_items = []
+
+            for item in serializer.validated_data:
+                try:
+                    with transaction.atomic():
+                        Event.objects.save(**item)
+                except Exception:
+                    failed_items.append((item, e))
+
+            if failed_items:
+                partial_error_response = {"error": "Some items could not be saved", "failed_items": failed_items}
+                headers = self.get_success_headers(serializer.data)
+                return Response(partial_error_response, status=status.HTTP_206_PARTIAL_CONTENT, headers=headers)
```

Comment:
it looks like 206 is really intended for "range" requests. Since that isn't what this is, I think it's fine to use a more standard response like `200`, especially since that is still distinguishable from a full success.

Reply:
Do you still plan to update this?

Reply:
Yeah, I have updated it to 201